.eslintrc

```diff
@@ -0,0 +1,105 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
+  "no-mixed-spaces-and-tabs": [2, "smart-tabs"],
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "jasmine": true
+  },
+  "rules": {
+    "array-callback-return": 0,
+    "max-len": ["error", 80],
+    "one-var": 0,
+    "no-constant-condition": 1,
+    "one-var-declaration-per-line": 2,
+    "new-cap": 0,
+    "consistent-return": 0,
+    "no-param-reassign": 0,
+    "comma-dangle": 0,
+    "react/jsx-uses-vars": 1,
+    "no-unused-vars": "warn",
+    "space-before-function-paren": 0,
+    "no-tabs": 0,
+    "no-use-before-define": 0,
+    "curly": [
+      "error",
+      "multi-line"
+    ],
+    "import/no-unresolved": [
+      0,
+      {
+        "commonjs": true
+      }
+    ],
+    "import/extensions": [
+      0,
+      "never",
+      {
+        "json": "always"
+      }
+    ],
+    "import/newline-after-import": 0,
+    "class-methods-use-this": ["error", {
+      "exceptMethods": [
+        "render"
+      ]
+    }],
+    "no-shadow": [
+      "error",
+      {
+        "allow": [
+          "req",
+          "res",
+          "err"
+        ]
+      } 
+    ],
+    "valid-jsdoc": [
+      "error",
+      {
+        "requireReturn": true,
+        "requireReturnType": true,
+        "requireParamDescription": false,
+        "requireReturnDescription": true
+      }
+    ],
+    "require-jsdoc": [
+      "error",
+      {
+        "require": {
+          "FunctionDeclaration": true,
+          "MethodDefinition": true,
+          "ClassDeclaration": true
+        }
+      }
+    ]
+  },
+  "settings": {
+    "import/resolver": "node"
+  },
+  "globals": {
+    "jest": true,
+    "no-unused-vars": false,
+    "localStorage": true,
+    "location": true,
+    "document": true,
+    "window": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true,
+      "experimentalObjectRestSpread": true
+    }
+  },
+  "plugins": [
+    "react",
+    "jsx-a11y",
+    "import"
+  ]
+}
```

Comment:
Kindly add a newline here and also do the same for most files here as they do not have a newline at end of file.

Reply:
I have fixed this